clang/lib/Headers/aiev2_scl2vec.h

```diff
@@ -51,11 +51,10 @@ INTRINSIC(v16accfloat)
 shiftx(v16accfloat a, v16accfloat b, int step, int shift) {
   return __builtin_aiev2_vshift_I512_I512(a, b, step, shift);
 }
-#if 0
```

Comment:
Curious: What was blocking the definition of those intrinsics before?

Reply:
I think initially we didn't define all these floating point vector types. Since those were added a while back, we could easily enable them now